packages/webapp/src/members/components/member-social-links.tsx

```diff
@@ -28,65 +28,68 @@ const urlify = (address: string) => {
+    <div className="flex flex-col flex-wrap max-h-24 text-sm">
+        <SocialButton
+            handle={member.account}
+            icon={EosCommunityIcon}
+            color="black"
+            size={4}
+            href={`${blockExplorerAccountBaseUrl}/${member.account}`}
+        />
+        {member.socialHandles.eosCommunity && (
             <SocialButton
-                handle={member.account}
-                icon={EosCommunityIcon}
+                handle={member.socialHandles.eosCommunity}
+                icon={IoChatbubblesOutline}
                 color="black"
-                href={`${blockExplorerAccountBaseUrl}/${member.account}`}
+                size={4}
+                href={`https://eoscommunity.org/u/${member.socialHandles.eosCommunity}`}
             />
-            {member.socialHandles.eosCommunity && (
-                <SocialButton
-                    handle={member.socialHandles.eosCommunity}
-                    icon={IoChatbubblesOutline}
-                    color="gray"
-                    href={`https://eoscommunity.org/u/${member.socialHandles.eosCommunity}`}
-                />
-            )}
-            {member.socialHandles.blog && (
-                <SocialButton
-                    handle="Blog"
-                    icon={FaBlogger}
-                    color="yellow"
-                    href={urlify(member.socialHandles.blog)}
-                />
-            )}
-        </div>
-        <div className="mt-2 sm:mt-0 sm:ml-4 space-y-2">
-            {member.socialHandles.twitter && (
-                <SocialButton
-                    handle={member.socialHandles.twitter}
-                    icon={FaTwitter}
-                    color="blue"
-                    href={`https://twitter.com/${member.socialHandles.twitter}`}
-                />
-            )}
-            {member.socialHandles.telegram && (
-                <SocialButton
-                    handle={member.socialHandles.telegram}
-                    icon={FaTelegram}
-                    color="blue"
-                    href={`https://t.me/${member.socialHandles.telegram}`}
-                />
-            )}
-            {member.socialHandles.linkedin && (
-                <SocialButton
-                    handle={member.socialHandles.linkedin}
-                    icon={FaLinkedin}
-                    color="blue"
-                    href={`https://www.linkedin.com/in/${member.socialHandles.linkedin}`}
-                />
-            )}
-            {member.socialHandles.facebook && (
-                <SocialButton
-                    handle={member.socialHandles.facebook}
-                    icon={FaFacebook}
-                    color="indigo"
-                    href={`https://facebook.com/${member.socialHandles.facebook}`}
-                />
-            )}
-        </div>
+        )}
+        {member.socialHandles.blog && (
+            <SocialButton
+                handle="Blog"
```

Comment:
now that I'm thinking, I believe we should change this icon to a link icon (eg: 
![image](https://user-images.githubusercontent.com/79721020/117133889-d5379a00-ad72-11eb-9558-352e10a62678.png)) and then replace the Blog word with the url... thoughts? not for this PR too.

Reply:
Yeah, I agree.